ocs_ci/templates/app-pods/cirros.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: 'v1'
+kind: 'DeploymentConfig'
+metadata:
+  name: 'cirrospod'
+spec:
+  template:
+    metadata:
+      labels:
+        name: 'cirrospod'
+    spec:
+      restartPolicy: 'Always'
+      volumes:
+      - name: 'cirros-vol'
+        persistentVolumeClaim:
+          claimName: 'pvcClaimCirros'
+      containers:
+      - name: 'cirros'
+        image: 'cirros'
+        imagePullPolicy: 'IfNotPresent'
+        volumeMounts:
+        - mountPath: '/mnt'
+          name: 'cirros-vol'
+        command: ['sh']
+        args:
+        - '-ec'
+        - 'while true; do
+               (mount | grep /mnt) && (head -c 1000 < /dev/urandom > /mnt/random-data.log) || exit 1;
```

Comment:
you are doing the same in code, why is this needed again?

Reply:
@vasukulkarni This yaml file attaches the PVC and run dd IO continuously till the POD is deleted. Also this pod has liveliness probe enabled (line 30), so deleting this pod is actually respin of POD and after respin again IO will continue.

Reply:
@ramkiperiy thanks for explanation.